### PR TITLE
Mint DOIs for remaining TNs and added README files for how to cite

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
Hi guys, do not forget to review this PR, to get done with the task of getting DOIs to TNs. Thanks.

**Comment by eduardo-rodrigues** [comment]
Guys, happy with the PR to finalise the idea of getting DOIs to TN? Thanks.

**Comment by graeme-a-stewart** [comment]
Not forgotten - will review soon!

File: HSF-TN/2015-01/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# HSF Technical Note HSF-TN-2015-01
+
+How to cite:
+
+McNab, A. (2015, September 30). HSF Technical Notes policy. HSF-TN-2015-01.
+Zenodo. [http://doi.org/10.5281/zenodo.1469623](http://doi.org/10.5281/zenodo.1469623)
```
Comment (graeme-a-stewart) [inline HSF-TN/2015-01/README.md]: Just for best practice I would make this the `https` versions.

File: HSF-TN/2016-01/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# HSF Technical Note HSF-TN-2016-01
+
+How to cite:
+
+Jouvin, M., Harvey, J., McNab, A., Sexton-Kennedy, E., & Wenaus, T. (2016, February 16). Software Licence Agreements HSF Policy Guidelines. HSF-TN-2016-01.
+Zenodo. [http://doi.org/10.5281/zenodo.1469636](http://doi.org/10.5281/zenodo.1469636)
```
Comment (graeme-a-stewart) [inline HSF-TN/2016-01/README.md]: ditto

File: HSF-TN/2016-02/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# HSF Technical Note HSF-TN-2016-02
+
+How to cite:
+
+Alef, M., Cass, T., Keijser, J. J., McNab, A., Roiser, S., Schwickerath, U., & Sfiligoi, I. (2016, February 4). Machine/Job Features. HSF-TN-2016-02.
+Zenodo. [http://doi.org/10.5281/zenodo.1469629](http://doi.org/10.5281/zenodo.1469629)
```
Comment (graeme-a-stewart) [inline HSF-TN/2016-02/README.md]: ditto

File: HSF-TN/2016-03/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# HSF Technical Note HSF-TN-2016-03
+
+How to cite:
+
+L. Sexton-Kennedy, B. Hegner, & B. Viren. (2016, March 23). HSF Packaging Working Group Report. HSF-TN-2016-03.
+Zenodo. [http://doi.org/10.5281/zenodo.1472340](http://doi.org/10.5281/zenodo.1472340)
```
Comment (graeme-a-stewart) [inline HSF-TN/2016-03/README.md]: ditto

File: HSF-TN/2016-04/README.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# HSF Technical Note HSF-TN-2016-04
+
+How to cite:
+
+McNab, A. (2016, October 4). Vacuum Platform. HSF-TN-2016-04.
+Zenodo. [http://doi.org/10.5281/zenodo.1469634](http://doi.org/10.5281/zenodo.1469634)
```
Comment (graeme-a-stewart) [inline HSF-TN/2016-04/README.md]: ditto

File: HSF-TN/2018-01/README.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# HSF Technical Note HSF-TN-2018-01
+
+How to cite:
+
+B. Hegner. (2018, June 22). HSF Platform Naming Conventions - A Proposal. HSF-TN-2018-01.
+Zenodo. [http://doi.org/10.5281/zenodo.1472351](http://doi.org/10.5281/zenodo.1472351)
```
Comment (graeme-a-stewart) [inline HSF-TN/2018-01/README.md]: ditto

**Comment by eduardo-rodrigues** [comment]
Hi @graeme-a-stewart, well spotted! Sure, makes complete sense. HTTPS now used everywhere.

**Pull request merged**